acceptance/setup/aio/010_Install.rb

```diff
@@ -12,6 +12,15 @@ def install_from_local(hosts, package)
       scp_to(agent, package, tmpdir)
     end
 
+    if agent[:platform].match(/(?:el-7|redhat-7)/)
+      step "Upgrade openssl package...(" + agent[:platform] + ")" do
```

Comment:
Your use of blocks here is strange. I'd suggest putting the `yum install` inside the block.

Reply:
Same below as well.